intera_interface/src/motion_interface/motion_trajectory.py

```diff
@@ -0,0 +1,183 @@
+#! /usr/bin/env python
+
+# Copyright (c) 2016-2017, Rethink Robotics Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import csv
+import yaml
+import rospy
+import rosbag
+import actionlib
+from motion_msgs.msg import (
+    Trajectory,
+    TrajectoryOptions,
+    Waypoint
+)
+import os.path
+from intera_core_msgs.msg import (
+    JointLimits
+)
+from sensor_msgs.msg import (
+    JointState
+)
+from motion_interface.motion_controller_action_client import (
+    MotionControllerActionClient
+)
+from motion_interface.motion_waypoint import MotionWaypoint
+from motion_interface.motion_waypoint_options import MotionWaypointOptions
+from motion_interface.utility_functions import ensure_path_to_file_exists
+from copy import deepcopy
+from rospy_message_converter import message_converter
+
+
+class MotionTrajectory(object):
+
+    @staticmethod
+    def get_default_joint_names():
+        """
+        @return: a names for sawyer's primary joints
+        """
+        return ['right_j0', 'right_j1', 'right_j2', 'right_j3',
```

Comment:
Grab these from the [`Limb`](https://github.com/RethinkRobotics/intera_sdk/blob/master/intera_interface/src/intera_interface/limb.py#L72-L82) or [`RobotParams`](https://github.com/RethinkRobotics/intera_sdk/blob/master/intera_interface/src/intera_interface/robot_params.py#L84) interface - they are retrieved from the parameter server.